### DISCUSSION TITLE
Firefox Pinned Shortcuts on New Tab

### DISCUSSION THREAD

**Discussion by kerbyyy** [yokoffing/Betterfox#207] [Q&A] [answered]
Every time I close and re-open Firefox, my pinned websites on the New Tab keeps disabling on its own and I have to re-enable it to see my pinned websites again. Is there a way to just let it stay each time I re-open Firefox and open a new tab?

**Comment by yokoffing** [comment, marked as answer]
The pref that is disabling pinned shortcuts is from Peskyfox:

https://github.com/yokoffing/Betterfox/blob/1a06cb1a47e6d9f388c39bbac6f1c95d5200e895/Peskyfox.js#L217

So you just need to add this to the bottom of your `user.js` file:

`user_pref("browser.newtabpage.activity-stream.feeds.topsites", true); // restore Top Sites on New Tab page `

**Reply by yokoffing** [reply]
I've also added this to the [Overrides sticky](https://github.com/yokoffing/Betterfox/issues/87).

**Reply by kerbyyy** [reply]
Yay! Thank you for the help.
